io.gitlab.construo.construo.json

```diff
@@ -0,0 +1,31 @@
+{
+    "app-id": "io.gitlab.construo.construo",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "rename-icon": "construo",
+    "rename-desktop-file": "construo.x11.desktop",
+    "rename-appdata-file": "construo.appdata.xml",
+    "finish-args": [
+        "--device=all",
```

Comment:
oh, I forgot to ask - what do you need device=all for, access to joysticks etc?

Once that's answered this LGTM